azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/factories/BaseViewModelFactory.kt

```diff
@@ -10,4 +11,8 @@ internal open class BaseViewModelFactory constructor(
     val warningsViewModel by lazy {
         PermissionWarningViewModel(store::dispatch)
     }
+
+    val errorInfoViewModel by lazy {
```

Comment:
what happened to snackBarViewModel? is it still used elsewhere?

Reply:
i.e. I see a rename in last file, and an addition to this file, but not a rename/removal from this file.

Reply:
nvm, I see deletion in next file.